examples/plot_line_testing.py

```diff
@@ -0,0 +1,134 @@
+"""
+Testing Line Plots
```

Comment:
Oh goodness, missed one. Sorry @nkorinek  This needs a more complete title: "Test Line Plots with Matplotcheck"